src/lib.rs

```diff
@@ -86,6 +88,29 @@ pub fn run(
 
     log::info!("Loading book summary: {}", mdbook_summary_path.display());
     let summary_src = std::fs::read_to_string(&mdbook_summary_path)?;
+
+    // Filter out any slides given in the `skip_slides` toml array entry
+    let summary_src = match book_skip_slides {
```

Comment:
I think this match could `if let Some(skip_list) { summary_src = summary_src.do_stuff().collect::<String>() }`. That might read a little easier.

Reply:
Ah, agree, `if let Some(___)` is preferable here.